### DISCUSSION TITLE
Jest startup time

### DISCUSSION THREAD

**Discussion by olaj** [vercel/next.js#13503] [Help] [answered]
The tests in my solution is kind of slow. And especially the startup time from that you start the test to it actually starts running is a bit irritating. It's usually like 5-10 sec.

Anyway, i wanted to test a standard project with a similar setup as mine and cloned

https://github.com/vercel/next.js/tree/canary/examples/with-typescript-eslint-jest

When i run a test there it's like 5 sec the first run and then 2-3 the second. Is this expected? Is there anyway to speed this up? I read a lot of posts about this but nothing has helped so far.

So i thought i asked about this example project to see if the startup time is expected and if anyone know how i can improve this?

**Comment by mattcarlotta** [comment, marked as answer]
Expected. There's not much you can do if your computer/laptop is older. Take a look at this [issue](https://stackoverflow.com/questions/61862533/why-does-npm-test-take-so-long-on-only-one-test), as it describes the same problem.
